arm-recoveryservices/compositeRecoveryServicesClient.json

```diff
@@ -0,0 +1,12 @@
+{
+  "info": {
+    "title": "RecoveryServicesClient",
+    "description": "Composite Swagger for Recovery Services Client"
+  },
+  "documents": [
+    "./2015-11-10/swagger/replicationusages.json",
```

Comment:
So, I missed this earlier. You cannot have different version references like so. What you want to do here is keep the 2015-11-10/swagger/replicationusages.json file as it is and create a new file 2016-06-01/swagger/replicationusages.json and have it referenced here.